src/main/java/org/folio/services/acquisitions/AcquisitionService.java

```diff
@@ -38,11 +37,12 @@ public void createAcquisitionsUnit(AcquisitionsUnit acquisitionsUnit, Context ve
       .onSuccess(entity -> {
         log.info("AcquisitionService with id {} created", acquisitionsUnit.getId());
         asyncResultHandler.handle(Future.succeededFuture(
-          AcquisitionsUnitsStorage.PostAcquisitionsUnitsStorageUnitsResponse.respond201WithApplicationJson(entity, headersFor201())));
+          PostAcquisitionsUnitsStorageUnitsResponse.respond201WithApplicationJson(
+            entity, PostAcquisitionsUnitsStorageUnitsResponse.headersFor201())));
       })
       .onFailure(throwable -> {
         log.error("AcquisitionService creation with id {} failed", acquisitionsUnit.getId(), throwable);
-        asyncResultHandler.handle(buildErrorResponse(throwable));
+        asyncResultHandler.handle(ResponseUtil.buildErrorResponse(throwable));
```

Comment:
I see in other places we use static import for buildErrorResponse, I think lets use the same to have standartized approach and code

Reply:
Now we have a killing combination: we use static imports and method with name `buildErrorResponse` is in more than one classes. That's one reason we there are recommendations to not use static imports at all.